lib/journaled/enqueue.rb

```diff
@@ -7,7 +7,7 @@ def enqueue!(*args)
     private
 
     def delayed_job_enqueue(*args, **opts)
-      Delayed::Job.enqueue(*args, **opts.reverse_merge(priority: Journaled.job_priority))
+      Delayed::Job.enqueue(*args, **opts.with_defaults(priority: Journaled.job_priority))
```

Comment:
These are aliased methods, I just like how `with_defaults` reads in this context. Tho might want to revert cause this is just git churn, plus I think `with_defaults` is only available in certain versions of Ruby

Will likely revert